ee/docker-compose.ch.hobby.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+
+services:
+    db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
+        ports:
+            - '5432:5432'
```

Comment:
I'm not sure what would be the documentation to run this (maybe we will suggest to add some firewall rule?) but I don't think we should expose PostgreSQL to the world, it should only be accessible to `localhost` (or maybe even to the linked containers).